openhtf/core/test_state.py

```diff
@@ -130,7 +146,53 @@ def test_api(self):
                 running_phase_state.attachments,
                 running_phase_state.attach,
                 running_phase_state.attach_from_file,
-                self.notify_update))
+                self.get_measurement,
+                self.get_attachment,
+                self.notify_update,))
+
+  def get_measurement(self, measurement_name):
+    """Get a copy of a measurement value from current or previous phase.
+
+    Args:
+      measurement_name: str of the measurement name
+    Returns:
+      an ImmutableMeasurement
+    """
+    # The framework ignores measurements from SKIP and REPEAT phases
+    ignore_outcomes = set([test_record.PhaseOutcome.SKIP,])
```

Comment:
`{test_record.PhaseOutcome.SKIP}` i think works?  also maybe make this IGNORED_OUTCOMES, though not sure you need a set() since it only has one thing in it now?

Reply:
Done.  Would rather keep variable local since it only make sense in this context and not at the level.  i.e test_state.IGNORED_OUTCOMES would imply that they are ignored at test_state level.